### ISSUE TITLE
Failure to download vcpkg packages within Github actions.

### ISSUE THREAD

**Issue by ni4** [microsoft/vcpkg#41339]
**Describe the bug**

Starting from yesterday Github Windows-2022 fails to download misc packages (yesterday it was Botan, today we have bzip2).
Could it be something on our side, or it's some network failure on the vcpkg side?

```
Building bzip2[core,tool]:x64-windows@1.0.8#6...
-- Downloading https://sourceware.org/pub/bzip2/bzip2-1.0.8.tar.gz;https://www.mirrorservice.org/sites/sourceware.org/pub/bzip2/bzip2-1.0.8.tar.gz -> bzip2-1.0.8.tar.gz...
[DEBUG] To include the environment variables in debug output, pass --debug-env
[DEBUG] Trying to load bundleconfig from C:\vcpkg\vcpkg-bundle.json
[DEBUG] Failed to open: C:\vcpkg\vcpkg-bundle.json
[DEBUG] Bundle config: readonly=false, usegitregistry=false, embeddedsha=nullopt, deployment=Git, vsversion=nullopt
[DEBUG] Metrics enabled.
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Feature flag 'registries' unset
[DEBUG] Feature flag 'versions' unset
[DEBUG] Feature flag 'dependencygraph' unset
warning: Download failed -- retrying after 1000ms
warning: Download failed -- retrying after 2000ms
warning: Download failed -- retrying after 4000ms
warning: Download failed -- retrying after 1000ms
warning: Download failed -- retrying after 2000ms
warning: Download failed -- retrying after 4000ms
error: Missing bzip2-1.0.8.tar.gz and downloads are blocked by x-block-origin.
error: https://sourceware.org/pub/bzip2/bzip2-1.0.8.tar.gz: WinHttpSendRequest failed with exit code 10106
error: https://sourceware.org/pub/bzip2/bzip2-1.0.8.tar.gz: WinHttpSendRequest failed with exit code 10106
error: https://sourceware.org/pub/bzip2/bzip2-1.0.8.tar.gz: WinHttpSendRequest failed with exit code 10106
error: https://sourceware.org/pub/bzip2/bzip2-1.0.8.tar.gz: WinHttpSendRequest failed with exit code 10106
error: https://www.mirrorservice.org/sites/sourceware.org/pub/bzip2/bzip2-1.0.8.tar.gz: WinHttpSendRequest failed with exit code 10106
error: https://www.mirrorservice.org/sites/sourceware.org/pub/bzip2/bzip2-1.0.8.tar.gz: WinHttpSendRequest failed with exit code 10106
error: https://www.mirrorservice.org/sites/sourceware.org/pub/bzip2/bzip2-1.0.8.tar.gz: WinHttpSendRequest failed with exit code 10106
error: https://www.mirrorservice.org/sites/sourceware.org/pub/bzip2/bzip2-1.0.8.tar.gz: WinHttpSendRequest failed with exit code 10106
```

**Environment**
- OS: windows-2022

**Comment by Silverlan** [comment]
Same issue here when trying to install 7zip using vcpkg on a Windows GitHub runner.
My test workflow (https://github.com/Silverlan/test_vcpkg/blob/main/.github/workflows/vcpkg.yml):
```yml
name: Install 7zip via vcpkg

on:
  push:
    branches:
      - main
  pull_request:
    branches:
      - main

jobs:
  build:
    runs-on: windows-latest
    steps:
      - name: Install 7zip with vcpkg
        shell: bash
        run: |
          vcpkg install 7zip
```

Log:
```
2024-10-04T14:15:32.5495283Z Current runner version: '2.319.1'
2024-10-04T14:15:32.5515828Z ##[group]Operating System
2024-10-04T14:15:32.5516368Z Microsoft Windows Server 2022
2024-10-04T14:15:32.5516772Z 10.0.20348
2024-10-04T14:15:32.5517075Z Datacenter
2024-10-04T14:15:32.5517377Z ##[endgroup]
2024-10-04T14:15:32.5517667Z ##[group]Runner Image
2024-10-04T14:15:32.5518040Z Image: windows-2022
2024-10-04T14:15:32.5518381Z Version: 20240929.1.0
2024-10-04T14:15:32.5519261Z Included Software: https://github.com/actions/runner-images/blob/win22/20240929.1/images/windows/Windows2022-Readme.md
2024-10-04T14:15:32.5520523Z Image Release: https://github.com/actions/runner-images/releases/tag/win22%2F20240929.1
2024-10-04T14:15:32.5521274Z ##[endgroup]
2024-10-04T14:15:32.5521621Z ##[group]Runner Image Provisioner
2024-10-04T14:15:32.5522023Z 2.0.384.1
2024-10-04T14:15:32.5522314Z ##[endgroup]
2024-10-04T14:15:32.5535681Z ##[group]GITHUB_TOKEN Permissions
2024-10-04T14:15:32.5537280Z Contents: read
2024-10-04T14:15:32.5537694Z Metadata: read
2024-10-04T14:15:32.5538028Z Packages: read
2024-10-04T14:15:32.5538335Z ##[endgroup]
2024-10-04T14:15:32.5541007Z Secret source: Actions
2024-10-04T14:15:32.5541510Z Prepare workflow directory
2024-10-04T14:15:32.6236409Z Prepare all required actions
2024-10-04T14:15:32.6394860Z Getting action download info
2024-10-04T14:15:32.8431134Z Download action repository 'actions/checkout@v3' (SHA:f43a0e5ff2bd294095638e18286ca9a3d1956744)
2024-10-04T14:15:33.1144446Z Complete job name: build
2024-10-04T14:15:33.2342938Z ##[group]Run actions/checkout@v3
2024-10-04T14:15:33.2343432Z with:
2024-10-04T14:15:33.2343696Z   repository: Silverlan/test_vcpkg
2024-10-04T14:15:33.2344265Z   token: ***
2024-10-04T14:15:33.2344545Z   ssh-strict: true
2024-10-04T14:15:33.2344847Z   persist-credentials: true
2024-10-04T14:15:33.2345168Z   clean: true
2024-10-04T14:15:33.2345448Z   sparse-checkout-cone-mode: true
2024-10-04T14:15:33.2345813Z   fetch-depth: 1
2024-10-04T14:15:33.2346067Z   fetch-tags: false
2024-10-04T14:15:33.2346348Z   lfs: false
2024-10-04T14:15:33.2346586Z   submodules: false
2024-10-04T14:15:33.2346859Z   set-safe-directory: true
2024-10-04T14:15:33.2347172Z ##[endgroup]
2024-10-04T14:15:33.3553115Z Syncing repository: Silverlan/test_vcpkg
2024-10-04T14:15:33.3555153Z ##[group]Getting Git version info
2024-10-04T14:15:33.3555979Z Working directory is 'D:\a\test_vcpkg\test_vcpkg'
2024-10-04T14:15:33.4308659Z [command]"C:\Program Files\Git\bin\git.exe" version
2024-10-04T14:15:33.4508487Z git version 2.46.2.windows.1
2024-10-04T14:15:33.4550739Z ##[endgroup]
2024-10-04T14:15:33.4569503Z Temporarily overriding HOME='D:\a\_temp\1f4993f7-b464-4532-9067-23a7639b0132' before making global git config changes
2024-10-04T14:15:33.4570830Z Adding repository directory to the temporary git global config as a safe directory
2024-10-04T14:15:33.4577063Z [command]"C:\Program Files\Git\bin\git.exe" config --global --add safe.directory D:\a\test_vcpkg\test_vcpkg
2024-10-04T14:15:33.4780962Z Deleting the contents of 'D:\a\test_vcpkg\test_vcpkg'
2024-10-04T14:15:33.4785968Z ##[group]Initializing the repository
2024-10-04T14:15:33.4793121Z [command]"C:\Program Files\Git\bin\git.exe" init D:\a\test_vcpkg\test_vcpkg
2024-10-04T14:15:33.5694447Z Initialized empty Git repository in D:/a/test_vcpkg/test_vcpkg/.git/
2024-10-04T14:15:33.5725362Z [command]"C:\Program Files\Git\bin\git.exe" remote add origin https://github.com/Silverlan/test_vcpkg
2024-10-04T14:15:33.5947392Z ##[endgroup]
2024-10-04T14:15:33.5948378Z ##[group]Disabling automatic garbage collection
2024-10-04T14:15:33.5956633Z [command]"C:\Program Files\Git\bin\git.exe" config --local gc.auto 0
2024-10-04T14:15:33.6185954Z ##[endgroup]
2024-10-04T14:15:33.6186886Z ##[group]Setting up auth
2024-10-04T14:15:33.6198199Z [command]"C:\Program Files\Git\bin\git.exe" config --local --name-only --get-regexp core\.sshCommand
2024-10-04T14:15:33.6425069Z [command]"C:\Program Files\Git\bin\git.exe" submodule foreach --recursive "sh -c \"git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :\""
2024-10-04T14:15:35.2083063Z [command]"C:\Program Files\Git\bin\git.exe" config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2024-10-04T14:15:35.2321346Z [command]"C:\Program Files\Git\bin\git.exe" submodule foreach --recursive "sh -c \"git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :\""
2024-10-04T14:15:35.6136106Z [command]"C:\Program Files\Git\bin\git.exe" config --local http.https://github.com/.extraheader "AUTHORIZATION: basic ***"
2024-10-04T14:15:35.6363852Z ##[endgroup]
2024-10-04T14:15:35.6364684Z ##[group]Fetching the repository
2024-10-04T14:15:35.6378173Z [command]"C:\Program Files\Git\bin\git.exe" -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +d859304e7998d26df53e77f4448043e0140ae586:refs/remotes/origin/main
2024-10-04T14:15:36.6412673Z remote: Enumerating objects: 6, done.        
2024-10-04T14:15:36.6413277Z remote: Counting objects:  16% (1/6)        
2024-10-04T14:15:36.6413702Z remote: Counting objects:  33% (2/6)        
2024-10-04T14:15:36.6414140Z remote: Counting objects:  50% (3/6)        
2024-10-04T14:15:36.6414547Z remote: Counting objects:  66% (4/6)        
2024-10-04T14:15:36.6414948Z remote: Counting objects:  83% (5/6)        
2024-10-04T14:15:36.6417268Z remote: Counting objects: 100% (6/6)        
2024-10-04T14:15:36.6417838Z remote: Counting objects: 100% (6/6), done.        
2024-10-04T14:15:36.6418319Z remote: Compressing objects:  33% (1/3)        
2024-10-04T14:15:36.6419254Z remote: Compressing objects:  66% (2/3)        
2024-10-04T14:15:36.6421560Z remote: Compressing objects: 100% (3/3)        
2024-10-04T14:15:36.6422640Z remote: Compressing objects: 100% (3/3), done.        
2024-10-04T14:15:36.6424364Z remote: Total 6 (delta 0), reused 0 (delta 0), pack-reused 0 (from 0)        
2024-10-04T14:15:36.6763076Z From https://github.com/Silverlan/test_vcpkg
2024-10-04T14:15:36.6764409Z  * [new ref]         d859304e7998d26df53e77f4448043e0140ae586 -> origin/main
2024-10-04T14:15:36.6886648Z ##[endgroup]
2024-10-04T14:15:36.6888015Z ##[group]Determining the checkout info
2024-10-04T14:15:36.6889396Z ##[endgroup]
2024-10-04T14:15:36.6890190Z ##[group]Checking out the ref
2024-10-04T14:15:36.6897500Z [command]"C:\Program Files\Git\bin\git.exe" checkout --progress --force -B main refs/remotes/origin/main
2024-10-04T14:15:36.7240909Z branch 'main' set up to track 'origin/main'.
2024-10-04T14:15:36.7245323Z Switched to a new branch 'main'
2024-10-04T14:15:36.7268121Z ##[endgroup]
2024-10-04T14:15:36.7496551Z [command]"C:\Program Files\Git\bin\git.exe" log -1 --format='%H'
2024-10-04T14:15:36.7683800Z 'd859304e7998d26df53e77f4448043e0140ae586'
2024-10-04T14:15:36.8091968Z ##[group]Run vcpkg install 7zip
2024-10-04T14:15:36.8092423Z [36;1mvcpkg install 7zip[0m
2024-10-04T14:15:36.8107935Z shell: C:\Program Files\Git\bin\bash.EXE --noprofile --norc -e -o pipefail {0}
2024-10-04T14:15:36.8108477Z ##[endgroup]
2024-10-04T14:15:37.6306360Z Computing installation plan...
2024-10-04T14:15:39.3533176Z The following packages will be built and installed:
2024-10-04T14:15:39.3533837Z     7zip:x64-windows@24.08
2024-10-04T14:15:39.3534187Z   * vcpkg-cmake:x64-windows@2024-04-23
2024-10-04T14:15:39.3534716Z   * vcpkg-cmake-config:x64-windows@2024-05-23
2024-10-04T14:15:39.3535357Z Additional packages (*) will be modified to complete this operation.
2024-10-04T14:15:40.1077060Z Detecting compiler hash for triplet x64-windows...
2024-10-04T14:15:47.7028527Z Compiler found: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.41.34120/bin/Hostx64/x64/cl.exe
2024-10-04T14:15:47.8198312Z A suitable version of 7zip was not found (required v24.8.0).
2024-10-04T14:15:48.2523916Z Downloading https://github.com/ip7z/7zip/releases/download/24.08/7z2408-extra.7z
2024-10-04T14:15:48.2526049Z Extracting 7zip...
2024-10-04T14:15:48.2531186Z A suitable version of 7zr was not found (required v24.8.0).
2024-10-04T14:15:48.5833287Z Downloading https://github.com/ip7z/7zip/releases/download/24.08/7zr.exe
2024-10-04T14:15:48.7168759Z Restored 0 package(s) from C:\Users\runneradmin\AppData\Local\vcpkg\archives in 187 us. Use --debug to see more details.
2024-10-04T14:15:48.7170338Z Installing 1/3 vcpkg-cmake:x64-windows@2024-04-23...
2024-10-04T14:15:48.7171099Z Building vcpkg-cmake:x64-windows@2024-04-23...
2024-10-04T14:15:48.7485923Z -- Installing: C:/vcpkg/packages/vcpkg-cmake_x64-windows/share/vcpkg-cmake/vcpkg_cmake_configure.cmake
2024-10-04T14:15:48.7495335Z -- Installing: C:/vcpkg/packages/vcpkg-cmake_x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake
2024-10-04T14:15:48.7503720Z -- Installing: C:/vcpkg/packages/vcpkg-cmake_x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake
2024-10-04T14:15:48.7514085Z -- Installing: C:/vcpkg/packages/vcpkg-cmake_x64-windows/share/vcpkg-cmake/vcpkg-port-config.cmake
2024-10-04T14:15:48.7524379Z -- Installing: C:/vcpkg/packages/vcpkg-cmake_x64-windows/share/vcpkg-cmake/copyright
2024-10-04T14:15:48.7704151Z -- Performing post-build validation
2024-10-04T14:15:48.8284381Z Stored binaries in 1 destinations in 50.7 ms.
2024-10-04T14:15:48.8285330Z Elapsed time to handle vcpkg-cmake:x64-windows: 112 ms
2024-10-04T14:15:48.8286730Z vcpkg-cmake:x64-windows package ABI: eba2a29e23565c1127388386a46b3919463c346e5ba81740a57aa18015c7dad9
2024-10-04T14:15:48.8288214Z Installing 2/3 vcpkg-cmake-config:x64-windows@2024-05-23...
2024-10-04T14:15:48.8289002Z Building vcpkg-cmake-config:x64-windows@2024-05-23...
2024-10-04T14:15:48.8596707Z -- Installing: C:/vcpkg/packages/vcpkg-cmake-config_x64-windows/share/vcpkg-cmake-config/vcpkg_cmake_config_fixup.cmake
2024-10-04T14:15:48.8605923Z -- Installing: C:/vcpkg/packages/vcpkg-cmake-config_x64-windows/share/vcpkg-cmake-config/vcpkg-port-config.cmake
2024-10-04T14:15:48.8613613Z -- Installing: C:/vcpkg/packages/vcpkg-cmake-config_x64-windows/share/vcpkg-cmake-config/copyright
2024-10-04T14:15:48.8661489Z -- Skipping post-build validation due to VCPKG_POLICY_EMPTY_PACKAGE
2024-10-04T14:15:48.9162370Z Stored binaries in 1 destinations in 16.5 ms.
2024-10-04T14:15:48.9163266Z Elapsed time to handle vcpkg-cmake-config:x64-windows: 87.8 ms
2024-10-04T14:15:48.9164357Z vcpkg-cmake-config:x64-windows package ABI: bd808a43c2dfd433c1f7747aad957f51a5f820ba97c22788bf3395da5a24d309
2024-10-04T14:15:48.9165080Z Installing 3/3 7zip:x64-windows@24.08...
2024-10-04T14:15:48.9165456Z Building 7zip:x64-windows@24.08...
2024-10-04T14:15:48.9498864Z -- Downloading https://github.com/ip7z/7zip/archive/24.08.tar.gz -> ip7z-7zip-24.08.tar.gz...
2024-10-04T14:15:56.1173994Z [DEBUG] To include the environment variables in debug output, pass --debug-env
2024-10-04T14:15:56.1177591Z [DEBUG] Trying to load bundleconfig from C:\vcpkg\vcpkg-bundle.json
2024-10-04T14:15:56.1178789Z [DEBUG] Failed to open: C:\vcpkg\vcpkg-bundle.json
2024-10-04T14:15:56.1179984Z [DEBUG] Bundle config: readonly=false, usegitregistry=false, embeddedsha=nullopt, deployment=Git, vsversion=nullopt
2024-10-04T14:15:56.1181043Z [DEBUG] Metrics enabled.
2024-10-04T14:15:56.1181602Z [DEBUG] Feature flag 'binarycaching' unset
2024-10-04T14:15:56.1182180Z [DEBUG] Feature flag 'compilertracking' unset
2024-10-04T14:15:56.1182591Z [DEBUG] Feature flag 'registries' unset
2024-10-04T14:15:56.1182985Z [DEBUG] Feature flag 'versions' unset
2024-10-04T14:15:56.1183387Z [DEBUG] Feature flag 'dependencygraph' unset
2024-10-04T14:15:56.1183844Z warning: Download failed -- retrying after 1000ms
2024-10-04T14:15:56.1184288Z warning: Download failed -- retrying after 2000ms
2024-10-04T14:15:56.1184725Z warning: Download failed -- retrying after 4000ms
2024-10-04T14:15:56.1185332Z error: Missing ip7z-7zip-24.08.tar.gz and downloads are blocked by x-block-origin.
2024-10-04T14:15:56.1186134Z error: https://github.com/ip7z/7zip/archive/24.08.tar.gz: WinHttpSendRequest failed with exit code 10106
2024-10-04T14:15:56.1187184Z error: https://github.com/ip7z/7zip/archive/24.08.tar.gz: WinHttpSendRequest failed with exit code 10106
2024-10-04T14:15:56.1188224Z error: https://github.com/ip7z/7zip/archive/24.08.tar.gz: WinHttpSendRequest failed with exit code 10106
2024-10-04T14:15:56.1189753Z error: https://github.com/ip7z/7zip/archive/24.08.tar.gz: WinHttpSendRequest failed with exit code 10106
2024-10-04T14:15:56.1191001Z [DEBUG] D:\a\_work\1\s\src\vcpkg\base\downloads.cpp(1030): 
2024-10-04T14:15:56.1191707Z [DEBUG] Time in subprocesses: 36065us
2024-10-04T14:15:56.1192472Z [DEBUG] Time in parsing JSON: 2us
2024-10-04T14:15:56.1192991Z [DEBUG] Time in JSON reader: 0us
2024-10-04T14:15:56.1193503Z [DEBUG] Time in filesystem: 1027us
2024-10-04T14:15:56.1194031Z [DEBUG] Time in loading ports: 0us
2024-10-04T14:15:56.1194699Z [DEBUG] Exiting after 7.2 s (7044467us)
2024-10-04T14:15:56.1195121Z 
2024-10-04T14:15:56.1195688Z CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:32 (message):
2024-10-04T14:15:56.1196602Z       
2024-10-04T14:15:56.1197096Z       Failed to download file with error: 1
2024-10-04T14:15:56.1198028Z       If you are using a proxy, please check your proxy setting. Possible causes are:
2024-10-04T14:15:56.1198888Z       
2024-10-04T14:15:56.1199595Z       1. You are actually using an HTTP proxy, but setting HTTPS_PROXY variable
2024-10-04T14:15:56.1200954Z          to `https://address:port`. This is not correct, because `https://` prefix
2024-10-04T14:15:56.1202105Z          claims the proxy is an HTTPS proxy, while your proxy (v2ray, shadowsocksr
2024-10-04T14:15:56.1203388Z          , etc..) is an HTTP proxy. Try setting `http://address:port` to both
2024-10-04T14:15:56.1204264Z          HTTP_PROXY and HTTPS_PROXY instead.
2024-10-04T14:15:56.1204876Z       
2024-10-04T14:15:56.1205689Z       2. If you are using Windows, vcpkg will automatically use your Windows IE Proxy Settings
2024-10-04T14:15:56.1207178Z          set by your proxy software. See https://github.com/microsoft/vcpkg-tool/pull/77
2024-10-04T14:15:56.1208442Z          The value set by your proxy might be wrong, or have same `https://` prefix issue.
2024-10-04T14:15:56.1209375Z       
2024-10-04T14:15:56.1210048Z       3. Your proxy's remote server is out of service.
2024-10-04T14:15:56.1210732Z       
2024-10-04T14:15:56.1211586Z       If you've tried directly download the link, and believe this is not a temporary
2024-10-04T14:15:56.1212960Z       download server failure, please submit an issue at https://github.com/Microsoft/vcpkg/issues
2024-10-04T14:15:56.1214120Z       to report this upstream download server failure.
2024-10-04T14:15:56.1214759Z       
2024-10-04T14:15:56.1214995Z 
2024-10-04T14:15:56.1215190Z Call Stack (most recent call first):
2024-10-04T14:15:56.1217076Z   scripts/cmake/vcpkg_download_distfile.cmake:270 (z_vcpkg_download_distfile_show_proxy_and_fail)
2024-10-04T14:15:56.1218371Z   scripts/cmake/vcpkg_from_github.cmake:106 (vcpkg_download_distfile)
2024-10-04T14:15:56.1219343Z   ports/7zip/portfile.cmake:1 (vcpkg_from_github)
2024-10-04T14:15:56.1220069Z   scripts/ports.cmake:192 (include)
2024-10-04T14:15:56.1220478Z 
2024-10-04T14:15:56.1220486Z 
2024-10-04T14:15:56.1243022Z error: building 7zip:x64-windows failed with: BUILD_FAILED
2024-10-04T14:15:56.1244219Z See https://learn.microsoft.com/vcpkg/troubleshoot/build-failures?WT.mc_id=vcpkg_inproduct_cli for more information.
2024-10-04T14:15:56.1245077Z Elapsed time to handle 7zip:x64-windows: 7.2 s
2024-10-04T14:15:56.9437406Z Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
2024-10-04T14:15:56.9440950Z Then check for known issues at:
2024-10-04T14:15:56.9444326Z   https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+7zip
2024-10-04T14:15:56.9447316Z You can submit a new issue at:
2024-10-04T14:15:56.9451654Z   https://github.com/microsoft/vcpkg/issues/new?title=[7zip]+Build+error+on+x64-windows&body=Copy+issue+body+from+C%3A%2Fvcpkg%2Finstalled%2Fvcpkg%2Fissue_body.md
2024-10-04T14:15:56.9457791Z You can also submit an issue by running (GitHub CLI must be installed):
2024-10-04T14:15:56.9461335Z   gh issue create -R microsoft/vcpkg --title "[7zip] Build failure on x64-windows" --body-file C:/vcpkg/installed/vcpkg/issue_body.md
2024-10-04T14:15:56.9629742Z ##[error]Process completed with exit code 1.
2024-10-04T14:15:56.9900585Z Post job cleanup.
2024-10-04T14:15:57.3000359Z [command]"C:\Program Files\Git\bin\git.exe" version
2024-10-04T14:15:57.3255654Z git version 2.46.2.windows.1
2024-10-04T14:15:57.3368989Z Temporarily overriding HOME='D:\a\_temp\ec3101e0-20be-4835-9cba-ef93e171818f' before making global git config changes
2024-10-04T14:15:57.3370309Z Adding repository directory to the temporary git global config as a safe directory
2024-10-04T14:15:57.3377910Z [command]"C:\Program Files\Git\bin\git.exe" config --global --add safe.directory D:\a\test_vcpkg\test_vcpkg
2024-10-04T14:15:57.3656078Z [command]"C:\Program Files\Git\bin\git.exe" config --local --name-only --get-regexp core\.sshCommand
2024-10-04T14:15:57.3922352Z [command]"C:\Program Files\Git\bin\git.exe" submodule foreach --recursive "sh -c \"git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :\""
2024-10-04T14:15:57.8815733Z [command]"C:\Program Files\Git\bin\git.exe" config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2024-10-04T14:15:57.9087631Z http.https://github.com/.extraheader
2024-10-04T14:15:57.9136783Z [command]"C:\Program Files\Git\bin\git.exe" config --local --unset-all http.https://github.com/.extraheader
2024-10-04T14:15:57.9434960Z [command]"C:\Program Files\Git\bin\git.exe" submodule foreach --recursive "sh -c \"git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :\""
2024-10-04T14:15:58.3579894Z Cleaning up orphan processes

```

**Comment by danrbailey** [comment]
Same issue here with lz4, I'm not aware of anything that has changed on our end. Downloading manually via curl does work meaning GHA is able to access the internet, so looks like a vcpkg issue.
```
-- Downloading https://github.com/lz4/lz4/archive/v1.10.0.tar.gz -> lz4-lz4-v1.10.0.tar.gz...
warning: Download failed -- retrying after 1000ms
warning: Download failed -- retrying after 2000ms
warning: Download failed -- retrying after 4000ms
error: Missing lz4-lz4-v1.10.0.tar.gz and downloads are blocked by x-block-origin.
error: https://github.com/lz4/lz4/archive/v1.10.0.tar.gz: WinHttpSendRequest failed with exit code 10106
```

**Comment by dg0yt** [comment]
It is fixed in vcpkg, but not in Github Actions.
https://github.com/microsoft/vcpkg/issues/41199#issuecomment-2378255699, https://github.com/microsoft/vcpkg/releases/tag/2024.09.30

Alternative Mitigations:
- Don't use vcpkg from GHA.
- Don't use bash in Windows.
- Set environment variables.

**Comment by ni4** [comment]
@dg0yt Thanks for the detailed reply! Used the `Set envrionment variables` solution and it worked. So closing this.
